### PR TITLE
Add active model serializer

### PR DESCRIPTION
**Pull request by scottalexandra**
added serializers for poverty, migration, and education models
all tests passing

### CODE REVIEW

File: app/serializers/state_poverty_data_serializer.rb [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+             :female_below_poverty_35to44,
+             :female_below_poverty_45to54,
+             :female_below_poverty_55to64,
+             :female_below_poverty_65to74,
+             :female_below_poverty_over75,
+             :state,
+             :percent_below_poverty,
+             :percent_male_below_poverty,
+             :percent_female_below_poverty
+
```
Comment (scottalexandra) [inline app/serializers/state_poverty_data_serializer.rb:38]: only added three percentages... wasn't sure if we needed all of them... someone else can add them if they really want more percentages :)

File: spec/controllers/api/v1/education/data_controller_spec.rb [hunk 2]
```diff
@@ -3,17 +3,31 @@
 RSpec.describe Api::V1::Education::DataController, type: :controller do
 
   describe 'get index' do
+    before(:each) { create(:state_education_data) }
     it "returns list of all data as json" do
-      create(:state_education_data)
       get :index
-      data = JSON.parse(response.body)
+      data = JSON.parse(response.body)["data"]
+
```
Comment (scottalexandra) [inline spec/controllers/api/v1/education/data_controller_spec.rb:10]: I'm not sure if it was the addition of the active model serializers, but the format of the data changed... all nested under the { "data" => [ [ "id" => .... so had to modify the tests.

File: spec/controllers/api/v1/map/map_controller_spec.rb [hunk 3]
```diff
@@ -6,9 +6,9 @@
     it "returns json" do
       create(:map_state)
       get :index
-      data = JSON.parse(response.body)
+      data = JSON.parse(response.body)["map"]
+
```
Comment (scottalexandra) [inline spec/controllers/api/v1/map/map_controller_spec.rb:10]: The format of the map data also changed,  even though no serializer was added for this model.... not sure why?

File: spec/controllers/api/v1/poverty/data_controller_spec.rb [hunk 4]
```diff
@@ -3,17 +3,28 @@
...
+
+    it 'returns json from state-poverty-data-serializer' do
+      get :index
+      data = JSON.parse(response.body)["data"]
+
+      expect(response).to have_http_status(:success)
+      expect(data.first["percent_below_poverty"]).to eq(19.825)
+      expect(data.first["percent_male_below_poverty"]).to eq(9.75)
+      expect(data.first["percent_female_below_poverty"]).to eq(10.075000000000001)
+    end
```
Comment (scottalexandra) [inline spec/controllers/api/v1/poverty/data_controller_spec.rb:28]: Is there a way to control how many decimal places you print out? This number is entirely too long....

File: app/controllers/api/v1/education/data_controller.rb [hunk 5]
```diff
@@ -1,5 +1,6 @@
 class Api::V1::Education::DataController < ApplicationController
   def index
-    render json: StateEducationData.all
+    render json: StateEducationData.all,
```
Comment (Jwan622) [inline app/controllers/api/v1/education/data_controller.rb:3]: Alex mentioned that line 4 was unnecessary, she will change later.

**Pull request closed without merging**